package/firmware/ipq-wifi/Makefile

```diff
@@ -47,6 +47,7 @@ Don't install it for any other device!
   PREV_BOARD+=ipq-wifi-$(1)
 endef
 
+$(eval $(call generate-ipq-wifi-package,comfast_cf-e370ac,board-comfast_cf-e370ac.bin,COMFAST CF-E370AC))
```

Comment:
Please also upstream your boardfile. The process is explained on the [ath10k wiki](https://wireless.wiki.kernel.org/en/users/drivers/ath10k/boardfiles)

Reply:
I had send email to ath10k@lists.infradead.org, but it reject my email with "host bombadil.infradead.org[65.50.211.133] said: 550 http://david.woodhou.se/email.html (in reply to end of DATA command)", I don't know reason.

Reply:
Mailing list was not working a couple of times today.
Give it another go